src/content/docs/fr/upgrade-astro.mdx

```diff
@@ -0,0 +1,182 @@
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  # Mettre à jour Astro et les intégrations officielles en même temps
+  npx @astrojs/upgrade
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  # Mettre à jour Astro et les intégrations officielles en même temps
+  pnpm dlx @astrojs/upgrade
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  # Mettre à jour Astro et les intégrations officielles en même temps
+  yarn dlx @astrojs/upgrade
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+
+### Mise à jour manuelle
+
+Pour mettre à jour manuellement Astro et les intégrations vers leurs versions actuelles, utilisez la commande appropriée pour votre gestionnaire de paquets.
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  # Exemple : mise à niveau d'Astro avec des intégrations React et Tailwind
+  npm install astro@latest @astrojs/react@latest @astrojs/tailwind@latest
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  # Exemple : mise à niveau d'Astro avec des intégrations React et Tailwind
+  pnpm add astro@latest @astrojs/react@latest @astrojs/tailwind@latest
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  # Exemple : mise à niveau d'Astro avec des intégrations React et Tailwind
+  yarn add astro@latest @astrojs/react@latest @astrojs/tailwind@latest
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+### Installer un numéro de version spécifique
+
+Pour installer une [version d'Astro](https://www.npmjs.com/package/astro?activeTab=versions) ou des intégrations spécifiques, utilisez la commande appropriée de votre gestionnaire de paquets.
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  npm install astro@4.5.3 @astrojs/react@3.0.10
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  pnpm add astro@4.5.3 @astrojs/react@3.0.10
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  yarn add astro@4.5.3 @astrojs/react@3.0.10
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+
+## Mise à jour de la documentation
+
+Cette documentation est mise à jour à chaque [version mineure](#changements-mineurs) et [version majeure](#changements-majeurs). Lorsque de nouvelles fonctionnalités sont ajoutées, ou que l'utilisation existante change, la documentation sera mise à jour pour refléter le **comportement actuel d'Astro**. Si votre projet n'est pas mis à jour, vous pouvez remarquer que certains comportements ne correspondent pas à la documentation mise à jour.
+
+Les nouvelles fonctionnalités sont ajoutées aux documents avec le numéro de version spécifique dans lequel elles ont été ajoutées. Cela signifie que si vous n'avez pas mis à jour la dernière version d'Astro, certaines fonctionnalités documentées peuvent être indisponibles. Vérifiez toujours le numéro de version `Added in:` et assurez-vous que votre projet est mis à jour avant d'essayer d'utiliser les nouvelles fonctionnalités !
```

Comment:
Ça fait longtemps que je ne suis pas venu contribuer au projet, mais on laisse le `added in` en anglais ? Je ne sais plus du tout si ça a été traduis sur les pages ou pas.
```suggestion
Les nouvelles fonctionnalités sont ajoutées aux documents avec le numéro de version spécifique dans lequel elles ont été ajoutées. Cela signifie que si vous n'avez pas mis à jour la dernière version d'Astro, certaines fonctionnalités documentées peuvent être indisponibles. Vérifiez toujours le numéro de version `Ajouté à la version :` et assurez-vous que votre projet est mis à jour avant d'essayer d'utiliser les nouvelles fonctionnalités !
```

Reply:
Je ne sais pas, je n'ai pas trouvé de document qui mentionne ce genre de texte